File: Atividades/Lista03/1_Retangulo/Retangulo.java
public class Retangulo {
    private double base;
    private double altura;

    public Retangulo(double b, double a){
        this.base = b;
        this.altura = a;
    }

    public String ToString(){
        return "Retangulo { Base: "+ base + ", Altura: " + altura + "}";
    }

    public double getbase(){
        return this.base;
    }
    public double getaltura(){
        return this.altura;
    }
    public void setbase(double b){
        this.base = b;
    }
    public void setaltura(double a){
        this.altura = a;
    }

    public double CalcArea(){
        double area = getbase() * getaltura();
        return area;
    }
    public double CalcDiagonal(){
        double diagonal = Math.sqrt((getbase()*getbase())+(getaltura()*getaltura()));
        return diagonal;
    }
}

File: Aula/Aula12_Streamlit/retangulo.py
import math

class Retangulo:
    def __init__(self, base, altura):
        self.__base = base
        self.__altura = altura

    def __str__(self):
        return f"Base = {self.__base} - Altura = {self.__altura} "
    
    def calc_area(self):
        return self.__base * self.__altura
    
    def calc_diagonal(self):
        return math.sqrt(self.__base**2 + self.__altura**2)